tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
+        self.name = JSON
+
+    def deserialize_header(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_header(self, obj):
+        if not obj:
+            obj = {}
+        return json.dumps(obj)
+
+    def deserialize_body(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_body(self, obj):
+        if not obj:
```

Comment:
What's up with the defaults here? 

`serialize_body(None) -> {}` (instead of "null"?)

`serialize_body(False) -> {}` (instead of "false"?)

`serialize_body(True) -> "true"` (instead of "{}"?)